various changes

tjk113 opened the pull request:
- added brief highlighting of function docs when accessed from the sidebar links
- clicking anywhere on the function buttons in the sidebar now works
- removed built-in modules from requirements.txt

Wade7wastaken commented:
I think im just gonna copy your changes and commit them myself cus i don't wanna deal with merge conflicts

tjk113 committed:
various changes:
- added brief highlighting of function docs when accessed from the sidebar links
- clicking anywhere on the function buttons in the sidebar now works
- removed built-in modules from requirements.txt

tjk113 committed:
(actually) removed built-in modules from requirements.txt

tjk113 committed:
fixed function link color
